sysvinit/openbsd/cloudinitlocal.tmpl

```diff
@@ -0,0 +1,40 @@
+## template:jinja
+#!/bin/ksh
+
+# PROVIDE: cloudinitlocal
+# REQUIRE: NETWORKING
+
+# After NETWORKING because we don't want staticroute to wipe
+# the route set by the DHCP client toward the meta-data server.
+
+daemon="cloud-init"
+daemon_execdir="{{prefix}}/bin"
+daemon_flags="init -l"
+daemon_rtable=0
+daemon_timeout=1
+daemon_user=root
+
+. /etc/rc.d/rc.subr
+
+rc_bg="YES" # (undefined or "YES")
+rc_usercheck="YES"
+
+rc_start() {
+    if test -e /etc/cloud/cloud-init.disabled; then
```

Comment:
Good that we check this. On other *nixes we also check [kernel commandline](https://cloudinit.readthedocs.io/en/latest/howto/disable_cloud_init.html#method-2-kernel-commandline). Is that doable on OpenBSD? Using a `sysctl kern.something` maybe?

It's nice to allow the user to disable without modifying the root filesystem.

Same comment on the other 3 init scripts.

Reply:
see ` daemon_execdir="{{prefix}}/bin" ` for example

Reply:
I looked this up initially (kernel options) , but I did not see a way to do this.  When googling it also said to have something like kern.options=cloud-init.disabled would require customizing the kernel.  So, that is why I left the kernel option check out.